Applications/spire/source/ui/title_bar.cpp

```diff
@@ -16,16 +20,18 @@ namespace {
   }
 }
 
-title_bar::title_bar(QWidget* parent)
-    : title_bar(QImage(), parent) {}
+title_bar::title_bar(QWidget* body, QWidget* parent)
+    : title_bar(QImage(), body, parent) {}
 
-title_bar::title_bar(const QImage& icon, QWidget* parent)
-    : title_bar(icon, icon, parent) {}
+title_bar::title_bar(const QImage& icon, QWidget* body, QWidget* parent)
+    : title_bar(icon, icon, body, parent) {}
 
 title_bar::title_bar(const QImage& icon, const QImage& unfocused_icon,
-    QWidget* parent)
+    QWidget* body, QWidget* parent)
     : QWidget(parent),
-      m_is_dragging(false) {
+      m_body(body),
```

Comment:
Members must be initialized in the order they are declared in the header file/class. That means is dragging must be initialized before body.